client/auth/api.js

```diff
@@ -174,14 +168,12 @@ define([
      *   @param {String} [config.email]
      *   Email address used to pre-fill into the account form,
      *   but the user is free to change it.
-     *   @param {String} [config.ui]
-     *   UI to present - `lightbox` or `redirect` - defaults to `redirect`
-     *   @param {Number} [options.zIndex]
+     *   @param {String} [config.ui="redirect"]
+     *   UI to present - `lightbox` or `redirect`
+     *   @param {Number} [config.zIndex=100]
      *   only used when `config.ui=lightbox`. The zIndex of the lightbox background.
-     *   @default 100
-     *   @param {String} [options.background]
+     *   @param {String} [config.background=rgba(0,0,0,0.5)]
```

Comment:
Renamed from `options.background` to `config.background`, otherwise I was seeing an odd method signature:
<img width="765" alt="authapi_-_fxa-relier-client" src="https://cloud.githubusercontent.com/assets/557895/9616289/3082b684-50b2-11e5-9311-b67fae259529.png">